app/models/exam_authorization_request.rb

```diff
@@ -0,0 +1,24 @@
+class ExamAuthorizationRequest < ApplicationRecord
+  belongs_to :exam
+  belongs_to :user
+  belongs_to :organization
+
+  enum status: %i[pending approved rejected]
+
+  before_save :set_default_status!
+  after_update :notify_user!
+
+  def try_authorize!
+    exam.authorize! user if approved?
+  end
+
+  private
+
+  def set_default_status!
```

Comment:
As this model is new and we have no records, we could add this default as a database default

Reply:
Great, the default is indeed in the migration, I copy-pasted this from somewhere else and wasn't really sure if it was necessary.